### PR TITLE
Implemented Landing Page

### LINKED ISSUE #21: Implement Home Page
## Requirements

- [x] Provide link to the sign in page
- [x] Provide link to the sign up page
- [x] Describes project

### PR DESCRIPTION
**Pull request by mcndl13**
The default landing page for the peer assessment website has been implemented as a temporary placeholder until the actual UI for the landing page is designed.

### CODE REVIEW

**Comment by JayC-SF** [comment]
Don't forget to fill the information on the right about the project, the assignee (which should be you) and also linking the issue associated with it!

Also next time make sure to create a branch from your issue instead of creating a branch without an ID. We can discuss about it another time :P

File: frontend/src/pages/home/HomePage.tsx [hunk 1]
```diff
@@ -1,30 +1,82 @@
...
+              <span className="font-black italic text-red-500">RateMyPeer</span>
+            </h1>
+            <p className="mt-6 text-lg leading-8 text-gray-600">
+              RateMyPeer is a Peer Assessment System for university team
+              projects is designed to allow students to evaluate the
+              contributions and performance of their teammates
             </p>
+            <div className="mt-10 flex items-center justify-center gap-x-6">
+              <a
+                href="/sign-in"
```
Comment (JayC-SF) [inline frontend/src/pages/home/HomePage.tsx]: I think here we're confused with the term login and sign-in, however they both mean the same. Doesn't matter which one we use but we have to make sure we don't confuse sign-up and login. 

I'll let you make the decision of which terms we're gonna use for the project when creating an account as first user and logging in to the app :)

File: frontend/src/pages/home/HomePage.tsx [hunk 2]
```diff
@@ -1,30 +1,82 @@
...
+                className="rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+              >
+                Sign Up
+              </a>
+              <a
+                href="/login"
```
Comment (JayC-SF) [inline frontend/src/pages/home/HomePage.tsx:67]: For example sign up means to create an account but the href is to `/login` so we gotta fix it. You can also change the route on the `AppRoutes.tsx` file if you need to :P

**Review by JayC-SF — APPROVED** [review]
Everything else looks good to me, just make sure to merge with squash and fill out the information on the side of the pull request good job!

File: frontend/src/pages/home/HomePage.tsx [hunk 3]
```diff
@@ -1,30 +1,82 @@
...
+  return (
+    <div className="bg-white">
+      <header className="absolute inset-x-0 top-0 z-50">
+        <nav
+          aria-label="Global"
+          className="flex items-center justify-between p-6 lg:px-8"
+        >
+          <div className="flex lg:flex-1">
+            <a href="/" className="-m-1.5 p-1.5">
+              <p className="font-black italic text-2xl"> RateMyPeer</p>
```
Comment (LamdaDev) [inline frontend/src/pages/home/HomePage.tsx]: Hey Mouhamed, great job so far! Can you rename all instances of RateMyPeer to RateMyPeers as I think that name is a better website name? Thanks!

File: frontend/src/pages/home/HomePage.tsx [hunk 4]
```diff
@@ -1,30 +1,82 @@
...
-            <button onClick={() => setCount((count) => count + 1)}>
-                count is {count}
-            </button>
-            <p>
-                Edit <code>src/App.tsx</code> and save to test HMR
+        <div className="mx-auto max-w-2xl py-32 sm:py-48 lg:py-56">
+          <div className="text-center">
+            <h1 className="text-4xl font-bold tracking-tight text-gray-900 sm:text-6xl">
+              Welcome to{" "}
+              <span className="font-black italic text-red-500">RateMyPeer</span>
```
Comment (LamdaDev) [inline frontend/src/pages/home/HomePage.tsx]: Same thing here!

**Review by LamdaDev — APPROVED** [review]
Overall, just rename RateMyPeer to RateMyPeers and fix the href comment that JC mentioned. LGTM!

File: frontend/src/pages/home/HomePage.tsx (hunk 1 above)
Comment (mcndl13) [reply frontend/src/pages/home/HomePage.tsx]: I changed it. We now have a Sign-Up page and a Login Page.

image

**Pull request merged**